test/unittests/test_enclosure.py

```diff
@@ -0,0 +1,16 @@
+import unittest
+
+from mock import patch
+from ovos_utils.messagebus import FakeBus
+
+
+class TestEnclosureAPI(unittest.TestCase):
+    from ovos_utils.enclosure.api import EnclosureAPI
+    bus = FakeBus()
+    api = EnclosureAPI(bus)
+    # TODO: Test api methods
+
+
+class TestMark1(unittest.TestCase):
+    # TODO should this be removed from ovos-utils?
```

Comment:
the utils here are meant to follow the defined bus api for the mk1 faceplate, but any client can consume or emit those bus calls, i think of this as the "robot face api"

skills need to import these classes if they provide animations and such, but should not import the mk1 PHAL plugin for this

in general bus api wrappers should live here if stuff in the wild is supposed to use them, just like we shouldnt require OCP to use the audio service bus api or core to use the IntentQuery api

Reply:
it has the standalone plugin, this is for skills that will interact with that plugin....

Reply:
oh, you mean refactoring mk1 PHAL to become a generic "robot face" plugin? I like that

could even be a class in OPM that mk1 subclasses then

i like that, out of scope for ovos-utils but it's a nice idea. the led encoding can maybe be extended too, will give it a think, i have a sensehat that will be a good test (8*8 rgb  vs 8 * 16 mono blue)

Reply:
Created separate issue and updated TODO comment